### PR TITLE
Adjustments before launch

### LINKED ISSUE #40: Hood adjustments before launch
This issue collects all the necessary tasks in `sbt-hood` before hood release.

- [x] Change sbt urls to goes from hood/sbt-hood to hood/sbt (this involves changes in the sbt-hood microsite if we choose that road).
- [x] Style the home sidebar button.

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This PR gives a new style to the `Hood home` sidebar button.
Captura de pantalla 2020-03-11 a las 15 13 38

Also modifies base urls to goes from `hood/sbt-hood` to `hood/sbt`.
Closes #40

### CODE REVIEW

File: project/ProjectPlugin.scala [hunk 1]
```diff
@@ -52,7 +52,7 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val micrositeSettings: Seq[Def.Setting[_]] = Seq(
       micrositeName := "SBT-Hood",
-      micrositeBaseUrl := "/sbt-hood",
+      micrositeBaseUrl := "/sbt",
```
Comment (BenFradet) [inline project/ProjectPlugin.scala]: was this intended?

Comment (AntonioMateoGomez) [reply project/ProjectPlugin.scala]: Yes it is @BenFradet , It was one of the things the Hood team wanted to change. Now the docs urls will be:
Graddle-Hood: `https://47deg.github.io/hood/gradle/`
SBT-Hood: `https://47deg.github.io/hood/sbt/`
That way keeps coherence.

Comment (BenFradet) [reply project/ProjectPlugin.scala]: Thanks :+1: didn't know about the `hood` prefix

Comment (AntonioMateoGomez) [reply project/ProjectPlugin.scala]: Ey @BenFradet I think I´m missing the `hood` prefix in the `micrositeBaseUrl` setting.
`micrositeBaseUrl := "/hood/sbt",`
I´ll change it in another PR.

**Pull request merged**